packages/app/README.md

```diff
@@ -10,15 +10,57 @@ This is the front-end for the Cypress App.
 3. Open chrome (or another browser)
 4. It will show the new Vite powered app 
 
```

Comment:
Nit: the doc says this is the front-end for the Cypress App, however the Launchpad and reporter packages also contain front-end for Cypress. Can we make this a bit more specific to what this package is accomplishing vs those packages?

Reply:
We should do that, but let's approach it another PR. I'd love to stop referring to this package as "app" when there are 4 packages (the ones you listed and frontend-shared) that comprise the App from a user perspective.

Reply:
It'd be great if we could actually co-locate these someday
```
packages/app
      src/
         reporter/
         launchpad/
         shared/
         ...app specific files

```

Reply:
The dream